composer.json

```diff
@@ -164,6 +166,10 @@
         "lint-php-cs": "@php vendor/bin/php-cs-fixer fix --verbose --diff --dry-run",
         "lint-composer": "@composer validate --no-check-publish --strict",
         "lint-twig": "@php bin/console lint:twig templates/",
+        "lint-xliff": "@php bin/console lint:xliff translations/",
```

Comment:
This may make sense if a project use xliff files. still if project not using it it can remove it.